src/test/java/it/pagopa/interop/probing/telemetry/writer/unit/consumer/TelemetryReceiverTest.java

```diff
@@ -0,0 +1,61 @@
+package it.pagopa.interop.probing.telemetry.writer.unit.consumer;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import java.io.IOException;
+import javax.validation.ConstraintViolationException;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.awspring.cloud.messaging.listener.SimpleMessageListenerContainer;
+import it.pagopa.interop.probing.telemetry.writer.consumer.TelemetryReceiver;
+import it.pagopa.interop.probing.telemetry.writer.dto.TelemetryDto;
+import it.pagopa.interop.probing.telemetry.writer.service.TimestreamService;
+
+@SpringBootTest
+@ExtendWith(MockitoExtension.class)
+public class TelemetryReceiverTest {
+
+  @InjectMocks
+  @Autowired
+  private TelemetryReceiver telemetryReceiver;
+
+  @Mock
+  private TimestreamService timestreamService;
+
+  @MockBean
+  private SimpleMessageListenerContainer simpleMessageListenerContainer;
+
+  @Autowired
+  private ObjectMapper mapper;
+
+
+  @Test
+  @DisplayName("The receiveMessage method of TelemetryReceiver class is tested.")
```

Comment:
test message should contain information about the context of the test, in this case something like
`The receiveMessage method of TelemetryReceiver handles valid message successfully`